schemas/2016-03-30/Microsoft.ContainerService.json

```diff
@@ -0,0 +1,229 @@
+                }
+              ],
+              "description": "Microsoft.ContainerService/containerServices - Orchestrator Type"
+            },
+            "masterProfile": {
+              "oneOf": [
+                {
+                  "$ref": "#/definitions/masterProfile"
+                },
+                {
+                  "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+                }
+              ],
+              "description": "Microsoft.ContainerService/containerServices - Master Profile"
+            },
+            "agentPoolProfiles": {
+              "oneOf": [
+                {
+                  "$ref": "#/definitions/agentPoolProfiles"
+                },
+                {
+                  "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+                }
+              ],
+              "description": "Microsoft.ContainerService/containerServices - Agent Pool Profiles"
+            },
+            "linuxProfile": {
+              "oneOf": [
+                {
+                  "$ref": "#/definitions/linuxProfile"
+                },
+                {
+                  "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+                }
+              ],
+              "description": "Microsoft.ContainerService/containerServices - Linux Profile"
+            }
+          },
+          "type": "object",
+          "required": [
+            "orchestratorProfile",
+            "masterProfile",
+            "agentPoolProfiles",
+            "linuxProfile"
+          ]
+        }
+      },
+      "required": [
+        "name",
+        "apiVersion",
+        "type",
+        "properties",
+        "location"
+      ],
+      "description": "Microsoft.ContainerService/containerServices"
+    }
+  },
+  "definitions": {
+    "id": {
+      "properties": {
+        "id": {
+          "type": "string"
+        }
+      },
+      "type": "object",
+      "required": [
+        "id"
+      ]
+    },
+    "orchestratorProfile": {
+      "properties": {
+        "orchestratorType": {
+          "type": "string"
+        }
+      },
+      "type": "object",
+      "required": [
+        "orchestratorType"
+      ]
+    },
+    "masterProfile": {
+      "properties": {
+        "count": {
+          "oneOf": [
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            },
+            {
+              "type": "integer"
+            }
+          ]
+        },
+        "dnsPrefix": {
+          "type": "string"
+        }
+      },
+      "type": "object",
+      "required": [
+        "count",
+        "dnsPrefix"
+      ]
+    },
+    "publicKey": {
```

Comment:
this seems not defining the element of an array but the properties of an array, the correct one is using "items".